### PR TITLE
Add config-resolver url 1.5

### PR DESCRIPTION
**Pull request by paigehargrave**

### CODE REVIEW

**Review by yianL — APPROVED** [review]
LGTM!

File: modules/ROOT/pages/config-mutual-auth.adoc [hunk 1]
```diff
@@ -0,0 +1,21 @@
+= Mutual Authentication with Anypoint Platform
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+With Anypoint Runtime Fabric version 1.3.60 or later, you can use the Runtime Fabric *Settings* page in Runtime Manager to enable mutual authentication for connectivity between the Runtime Fabric and Anypoint control plane. Mutual authentication improves the security of communication between Runtime Fabric and Anypoint Platform and is highly recommended. Runtime Fabric instances created after July 8, 2020 have the mutual authentication setting enabled by default.
```
Comment (rddheath) [inline modules/ROOT/pages/config-mutual-auth.adoc:6]: Is it "between the Runtime Fabric and Anypoint control planes" or "between the Runtime Fabric and the Anypoint control plane"?

Comment (rddheath) [inline modules/ROOT/pages/config-mutual-auth.adoc:6]: "between Runtime Fabric and Anypoint Platform"
But you wrote "Anypoint control plane" in the previous sentence.
Do you even need to repeat the names? Could you just say "between both"?

Comment (rddheath) [inline modules/ROOT/pages/config-mutual-auth.adoc:6]: "mutual authentication setting"
Whenever two words are used as a single adjective, place a hyphen between them: "mutual-authentication"

File: modules/ROOT/pages/config-mutual-auth.adoc [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+Before enabling mutual authentication, verify the following endpoint and port are reachable from all Runtime Fabric controller and worker nodes:
```
Comment (rddheath) [inline modules/ROOT/pages/config-mutual-auth.adoc:8]: "verify the following endpoint and port"
However, you list two endpoints and two ports, so why refer to them in the singular?

**Review by rddheath — APPROVED** [review]

**Pull request merged**